cluster/os-3.9.0-alpha.4/README.md

```diff
@@ -0,0 +1,47 @@
+# OpenShift 3.9.0-alpha.4 in ephemeral containers
+
+Provides a pre-deployed OpenShift Origin with version 3.9.0-alpha.4 purely in docker
+containers with qemu. The provided VMs are completely ephemeral and are
+recreated on every cluster restart. The KubeVirt containers are built on the
+local machine and are the pushed to a registry which is exposed at
+`localhost:5000`.
+
+## Bringing the cluster up
+
+You will need to add line to `/etc/hosts` only once.
+
+```bash
+echo "127.0.0.1 node01" >> /etc/hosts
```

Comment:
why modifying /etc/hosts required?

Reply:
I think it connects to the fact that OpenShift environment must have FQDN that resolvable by DNS, so it a hack to make it work without it.

Reply:
Is that to make osp-console redirects work? I am not sure I understand that.

Reply:
This way it worked for me without having to modify /etc/hosts:

```diff
     OPTIONS="-o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -i ${KUBEVIRT_PATH}cluster/vagrant.key -P 2201"
 
     # Copy oc tool
+    ${_cli} ssh --prefix $prefix node01 sudo cp /etc/origin/master/admin.kubeconfig ~vagrant/
+    ${_cli} ssh --prefix $prefix node01 sudo chown vagrant:vagrant ~vagrant/admin.kubeconfig
+    scp ${OPTIONS} vagrant@127.0.0.1:~vagrant/admin.kubeconfig ${KUBEVIRT_PATH}cluster/os-3.9.0-alpha.4/.kubeconfig
     scp ${OPTIONS} vagrant@127.0.0.1:/usr/local/bin/oc ${KUBEVIRT_PATH}cluster/os-3.9.0-alpha.4/.oc
     chmod u+x ${KUBEVIRT_PATH}cluster/os-3.9.0-alpha.4/.oc
 
     # Login to OpenShift
     export KUBECONFIG=${KUBEVIRT_PATH}cluster/os-3.9.0-alpha.4/.kubeconfig
-    ${KUBEVIRT_PATH}cluster/os-3.9.0-alpha.4/.oc login $(_main_ip):8443 --insecure-skip-tls-verify=true -u admin -p admin
+    ${KUBEVIRT_PATH}cluster/os-3.9.0-alpha.4/.oc config set-cluster node01:8443 --server=https://127.0.0.1:8443
+    ${KUBEVIRT_PATH}cluster/os-3.9.0-alpha.4/.oc config set-cluster node01:8443 --insecure-skip-tls-verify=true
 
     # Make sure that local config is correct
     prepare_config
```

Might also work with setting one or all of the following environment variables: `NO_PROXY`, `HTTPS_PROXY`, `HTTP_PROXY`.

Reply:
@rmohr Can you also connect to web console without line under /etc/hosts?

Reply:
And `oc login` creates `.kubeconfig` automatically so no need to download and edit it.

Reply:
Well but I didn't have to modify `/etc/hosts` this way.

Reply:
@cynepco3hahue  No, console authentication still fails